auto_dev/data/repo/templates/autonomy/install.sh

```diff
@@ -127,26 +139,27 @@ function install_poetry_deps() {
     local pip_executable
     local poetry_executable
     local host_poetry_executable
-    host_poetry_executable=$(echo -n $(poetry env info | grep Executable |head -n 1 | awk -F: '{ print $2 }') | xargs)
-    echo "Host poetry executable: $host_poetry_executable"
+    host_poetry_executable="$(poetry env info | grep Executable | head -n 1 | awk -F: '{ print $2 }' | xargs)"
+    echo "Host poetry executable: ${host_poetry_executable}"
     echo "Setting up new poetry environment..."
 
-    poetry env use $(which python)
-    poetry_executable=$(echo -n $(poetry env info | grep Executable |head -n 1 | awk -F: '{ print $2 }') | xargs)
-    echo "New poetry executable:   $poetry_executable"
-
+    python_path="$(command -v python)"
+    poetry env use "${python_path}"
+    poetry_executable="$(poetry env info | grep Executable | head -n 1 | awk -F: '{ print $2 }' | xargs)"
+    echo "New poetry executable:   ${poetry_executable}"
+    
     echo "Installing package dependencies via poetry..."
-    echo "Using poetry executable: $poetry_executable"
-    poetry install > /dev/null || exit 1
+    echo "Using poetry executable: ${poetry_executable}"
+    export PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring
+    poetry install --with dev > /dev/null || exit 1
     echo "Checking if aea is installed"
     poetry run aea --version
     echo "Done installing dependencies"
-
 }
 # Main execution
 
- 
-function set_env_file () {
+
+function set_env_file() {
```

Comment:
why did you rename all the files ahah

Reply:
to make them more standard and shell-agnostic, reverted in 79afc73e5c86ed64a8afa8c4da1870b7ba80c036